src/components/App/App.css

```diff
@@ -0,0 +1,11 @@
+header {
+  display: flex;
+}
+
+h1 {
+  color: #A10115;
+  font-size: 4em;
+  position: relative;
+  top: 83px;
+  left: -164px;
```

Comment:
love the look of the header :) I worry a bit about the specificity of the pixels but seems to be looking good on all the screen sizes.